Tools/DeployApps/BaasClient.cs

```diff
@@ -473,9 +475,9 @@ private async Task<BaasApp> CreateFlxApp(string name)
                 {
                     state = "enabled",
                     database_name = GetSyncDatabaseName(name),
-                    queryable_fields_names = new[] { "Int64Property", "GuidProperty", "DoubleProperty", "Int", "Guid", "Id", "PartitionLike" },
+                    queryable_fields_names = new[] { "Int64Property", "GuidProperty", "DoubleProperty", "Int", "Guid", "Id", "PartitionLike", "Differentiator" },
```

Comment:
Not a biggie, but maybe we should find another name for "differentiator"? We already use it for differentiating the apps, maybe it could create confusion

Reply:
We do the same in dart.

Reply:
I feel it's a reasonable name given that it differentiates things based on some criteria. Is it the best name - probably not, but I liked it better than `PartitionLike` 😄